src/ReaderScreen/components/autoScrollComponent.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { View, Text } from "react-native";
+import Slider from "@react-native-community/slider";
+import { useDispatch, useSelector } from "react-redux";
+import { Icon } from "@rneui/themed";
+import PropTypes from "prop-types";
+import colors from "../../common/colors";
+import styles from "../styles";
+import constant from "../../common/constant";
+import { setAutoScrollSpeed } from "../../common/actions";
+
+function AutoScrollComponent({ shabadID }) {
+  const [isPaused, togglePaused] = useState(true);
+  const { autoScrollSpeedObj } = useSelector((state) => state);
+  const currentSpeed = autoScrollSpeedObj[shabadID] || constant.DEFAULT_SPEED;
+  const dispatch = useDispatch();
+
+  const handleSpeed = (value) => {
+    dispatch(setAutoScrollSpeed(value, shabadID));
+    if (value === 0) togglePaused(true);
```

Comment:
use proper syntax of if with braces 
```js
if (condition) {
  ...
}
```


Are we using airbnb eslint style guide?